java/src/jmri/jmrit/display/layoutEditor/LayoutEditorChecks.java

```diff
@@ -346,7 +350,9 @@ private void setupCheckNonContiguousBlocksMenu() {
         // collect all contiguous blocks
         HashMap<String, List<Set<String>>> blockNamesToTrackNameSetMaps = new HashMap<>();
         for (LayoutTrack layoutTrack : layoutEditor.getLayoutTracks()) {
-            layoutTrack.checkForNonContiguousBlocks(blockNamesToTrackNameSetMaps);
+            if (layoutTrack != null) {
```

Comment:
I don’t think you need the null check here.  You are iterating over a collection of layoutTrack objects

Reply:
One of the LayoutTrack tests was crashing here with a NPE… I don't know how the null got in the list so I put in the check. If(/when) I get appveyor passing again I can spend time rooting out where the null came from… I won't close this as resolved.